cmd/login.go

```diff
@@ -56,9 +56,11 @@ func init() {
 	viper.AddConfigPath("/etc/mender-cli/")
 	viper.AddConfigPath("$HOME/")
 	viper.AddConfigPath(".")
-	err := viper.ReadInConfig()
-	if err != nil {
-		log.Info("No configuration file found")
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
+			log.Info(fmt.Sprintf("Failed to read config: %s", err))
+			os.Exit(1)
```

Comment:
Question:
Maybe we should log the error in this instance also?

Reply:
I don't see this as an error, because the parameters can also be set as command line args.

Reply:
My initial implementation was checking both cases resulting in a silent continuation if the config file was not pressent. I will add a log.Info in this case.